compute_tools/src/compute.rs

```diff
@@ -620,20 +626,24 @@ impl ComputeNode {
             });
         }
 
-        // Configure and start rsyslog for HIPAA if necessary
-        if let ComputeAudit::Hipaa = pspec.spec.audit_log_level {
-            let remote_endpoint = std::env::var("AUDIT_LOGGING_ENDPOINT").unwrap_or("".to_string());
-            if remote_endpoint.is_empty() {
-                anyhow::bail!("AUDIT_LOGGING_ENDPOINT is empty");
-            }
+        // If extended compute audit is enabled configure and start rsyslog
+        match pspec.spec.audit_log_level {
+            ComputeAudit::Hipaa | ComputeAudit::Full | ComputeAudit::FullWithParameters => {
+                let remote_endpoint =
```

Comment:
Seems like this could just be an Option<String> instead of a String

Reply:
I moved it to `configure_audit_rsyslog()` and changed to normal error instead of panic.